shared/kubernetes/provider.go

```diff
@@ -278,6 +278,36 @@ func (provider *Provider) GetMizuApiServerPodObject(opts *ApiServerOptions, moun
 				},
 			},
 		},
+		{
+			Name:            "basenine",
+			Image:           "ghcr.io/up9inc/basenine:v0.2.26",
```

Comment:
maybe this can be configuration option or at least const in the code so we don't need to search inside the function to see which image do we take

Reply:
@gadotroee the image is version locked. So user shouldn't be able to change this image. I put the information into `consts.go` with https://github.com/up9inc/mizu/commit/b44f304618cd52cbf0eace292366ea79b3da46a4